src/Ethereum.sol

```diff
@@ -47,65 +36,83 @@ library Ethereum {
+    address internal constant DAI                   = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
     address internal constant DAI_ATOKEN            = 0x4DEDf26112B3Ec8eC46e7E31EA5e123490B05B8B;
-    address internal constant DAI_IRM               = 0x7949a8Ef09c49506cCB1cB983317272dcf4170Dd;
-    address internal constant DAI_ORACLE            = 0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9;
     address internal constant DAI_STABLE_DEBT_TOKEN = 0xfe2B7a7F4cC0Fb76f7Fc1C6518D586F1e4559176;
     address internal constant DAI_DEBT_TOKEN        = 0xf705d2B7e92B3F38e6ae7afaDAA2fEE110fE5914;
 
+    address internal constant GNO                   = 0x6810e776880C02933D47DB1b9fc05908e5386b96;
+    address internal constant GNO_ATOKEN            = 0xbf13910620722D4D4F8A03962894EB3335Bf4FaE;
+    address internal constant GNO_STABLE_DEBT_TOKEN = 0x57a2957651DA467fCD4104D749f2F3684784c25a;
+    address internal constant GNO_DEBT_TOKEN        = 0x554265A713D6746A62d86A797254590784D436AA;
+
+    address internal constant RETH                   = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address internal constant RETH_ATOKEN            = 0x9985dF20D7e9103ECBCeb16a84956434B6f06ae8;
+    address internal constant RETH_STABLE_DEBT_TOKEN = 0xa9a4037295Ea3a168DC3F65fE69FdA524d52b3e1;
+    address internal constant RETH_DEBT_TOKEN        = 0xBa2C8F2eA5B56690bFb8b709438F049e5Dd76B96;
+
+    address internal constant SDAI                   = 0x83F20F44975D03b1b09e64809B757c47f942BEeA;
     address internal constant SDAI_ATOKEN            = 0x78f897F0fE2d3B5690EbAe7f19862DEacedF10a7;
-    address internal constant SDAI_IRM               = 0xeC4cf692c18E62159a39704Aa1Db82ca2306fF90;
-    address internal constant SDAI_ORACLE            = 0xb9E6DBFa4De19CCed908BcbFe1d015190678AB5f;
     address internal constant SDAI_STABLE_DEBT_TOKEN = 0xEc6C6aBEd4DC03299EFf82Ac8A0A83643d3cB335;
     address internal constant SDAI_DEBT_TOKEN        = 0xaBc57081C04D921388240393ec4088Aa47c6832B;
 
+    address internal constant USDC                   = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
     address internal constant USDC_ATOKEN            = 0x377C3bd93f2a2984E1E7bE6A5C22c525eD4A4815;
-    address internal constant USDC_IRM               = 0x0F1a9a787b4103eF5929121CD9399224c6455dD6;
-    address internal constant USDC_ORACLE            = 0x98e6BcBA7d5daFbfa4a92dAF08d3d7512820c30C;
     address internal constant USDC_STABLE_DEBT_TOKEN = 0x887Ac022983Ff083AEb623923789052A955C6798;
     address internal constant USDC_DEBT_TOKEN        = 0x7B70D04099CB9cfb1Db7B6820baDAfB4C5C70A67;
 
+    address internal constant USDT                   = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address internal constant USDT_ATOKEN            = 0x0Dae62F953Ceb2E969fB4dE85f3F9074fa920776;
+    address internal constant USDT_STABLE_DEBT_TOKEN = 0x529b6158d1D2992E3129F7C69E81a7c677dc3B12;
+    address internal constant USDT_DEBT_TOKEN        = 0x0F1a9a787b4103eF5929121CD9399224c6455dD6;
```

Comment:
These are in order: `0xe7dF13b8e3d6740fe17CBE928C7334243d86c92f`, `0x0Dae62F953Ceb2E969fB4dE85f3F9074fa920776` and `0x529b6158d1D2992E3129F7C69E81a7c677dc3B12`.

Reply:
Fixed